### PR TITLE
Polish the Expressions UI

### PR DESCRIPTION
**Pull request by jasonLaster**
Associated Issue: #1599

  - [x] Prevent duplicate expressions 
  - [x] Fix editing expressions 
  - [x] Re-evaluate on resume
  - [x] cursor on input should be "text"
  - [x] close button is off
  - [x] expressions should not expand by default
  - [ ] "Not Available" shouldn't be in quotes
  - [x] re-evaluate when a frame is selected
  - [x] can't remove last expression

### CODE REVIEW

File: src/actions/pause.js [hunk 1]
```diff
@@ -272,14 +280,18 @@ function deleteExpression(expression: Expression) {
...
  * @static
  */
-function evaluateExpressions(selectedFrameId: frameIdType) {
+function evaluateExpressions(frameId: frameIdType) {
   return async function({ dispatch, getState, client }: ThunkArgs) {
     for (let expression of getExpressions(getState())) {
+      if (!expression.input) {
+        continue;
+      }
+
```
Comment (jbhoosreddy) [inline src/actions/pause.js]: since we are no longer adding empty expressions, this check might not be needed here anymore, maybe?

**Comment by jbhoosreddy** [comment]
did leave a small comment there, but in either case this looks very good!

File: src/actions/pause.js (hunk 1 above)
Comment (jasonLaster) [inline src/actions/pause.js]: true, but I like being safe as it could blow up if we still hit it

Comment (jasonLaster) [inline src/actions/pause.js]: perhaps we could add a console.warn here as well so that we are notified?

Comment (jbhoosreddy) [inline src/actions/pause.js]: that sounds like a good idea.

**Pull request merged**